PaxConfig.cs

```diff
@@ -36,6 +37,9 @@ public class NetworkInterfaceConfig
     // The function that is called when traffic arrives on this interface.
     public string lead_handler {get; set;}
 
+    [DefaultValue(100)]
```

Comment:
Note to self: this is where the 100ms default value is set. Add a comment as per niksu's suggestion